### ISSUE TITLE
Deprecation notice after upgrading to Flutter 2.5

### ISSUE THREAD

**Issue by juzo** [appspector/flutter-plugin#48]
```
The plugin `appspector` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```

**Comment by Dimdron** [comment]
Thanks a lot for the report! We'll take a look as soon as possible!

**Comment by Dimdron** [comment]
It's included to 0.8.0 release
